instructor/common.py

```diff
@@ -1671,6 +1698,20 @@ def facs_histograms_edit(request, assignment, sample_prep, sp):
 
 
 @login_required
+def select_images(request):
+    mapping_id = request.POST.get('mapping_pk')
+    image_pk_list = request.POST.getlist('image_pk_list[]')
+    instance = get_object_or_404(models.MicroscopyImageMapping, pk=mapping_id)
+    for image_pk in image_pk_list:
+        selected_image = get_object_or_404(models.MicroscopyImage, pk=image_pk)
+
+        instance.images.add(selected_image)
+    instance.save()
+
+    return HttpResponse('complete')
+
+
+@login_required
 def submit_histogram(request):
```

Comment:
Do you need the `@assignment_selected` and `@check_assignment_owner` decorators here and on `select_images`?